lib/ui5Framework/npm/Registry.js

```diff
@@ -19,6 +25,9 @@ class Registry {
 	}
 	async requestPackageManifest(pkgName, version) {
 		const {pacote, pacoteOptions} = await this._getPacote();
+		// TODO: Pacote#manifest essentially installs and unpacks a package
```

Comment:
will this todo be part of this PR?

Reply:
We've decide we'd address those in a separate change